src/components/CardDetailsPage.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div id="card_details_page">
+      <div class="row justify-content-between mb-5 mr-5">
+        <logo :company_name="company_name" :app_name="app_name" style="padding-left:30px;margin-left: 35px"/>
```

Comment:
Consider keeping styling with "style" either in CSS files or at least as variables in data.